scripts/collection/tests/Invoke-CmtraceEvidenceCollection.Tests.ps1

```
@@ -101,6 +104,131 @@ Describe 'Intune evidence profile contracts' {
     }
 }
 
+Describe 'Optional array validation' {
+    It 'accepts a single-element optional array such as arguments: ["/status"]' {
+        $profile = New-TestCollectorProfile
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'profile' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)